src/controllers/users.js

```diff
@@ -38,4 +38,38 @@ export default class UsersController {
       return errorResponse(res, status.error, messages.signUp.error);
     }
   }
+
+  /**
+   * @method signInUser
+   * @description Method for user sign in
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @returns {object} response body object
+   */
+  static async signInUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await models.Users.findOne({ where: { email } });
+  
+      if (!user) {
+        return errorResponse(res, status.notfound, messages.signIn.notfound);
```

Comment:
Nice work @oyewoas , As a matter of security concerns wouldn't it be nice to send an authorization failure error rather than a not found error, doing this would prevent attackers from knowing the currently existing user's email of our app. When we send authorization failure error , the attacker wont be able to detect if its a none existing mail or wrong password